File: linear_algebra/Python/BTAT_BT1.py
m, n = map(int, input("Nhap so hang va so cot cua ma tran A: ").split())

A = []
print("Nhap cac phan tu cua ma tran A:")
for i in range(m):
    row = input(f"Dòng {i+1}: ").split()
    row_int = list(map(int, row))
    A.append(row_int)

print("Ma tran A la:")
for i in range(m):
    for j in range(n):
        print(A[i][j], end=" ")
    print()

File: linear_algebra/C++/BTAT_BT1.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int m, n;
    cout << "Nhap so hang va so cot cua ma tran A: ";
    cin >> m >> n;

    vector<vector<int>> A(m, vector<int>(n));

    cout << "Nhap cac phan tu cua ma tran A:\n";
    for (int i = 0; i < m; ++i)
        for (int j = 0; j < n; ++j)
            cin >> A[i][j];

    cout << "Ma tran A la:\n";
    for (int i = 0; i < m; ++i) {
        for (int j = 0; j < n; ++j)
            cout << A[i][j] << " ";
        cout << "\n";
    }

    return 0;
}